src/main/java/jeresources/entry/VillagerEntry.java

```diff
@@ -27,12 +27,11 @@ public VillagerEntry(VillagerProfession profession, Int2ObjectMap<VillagerTrades
     }
 
     public void addITradeLists(Int2ObjectMap<VillagerTrades.ITrade[]> tradesLists) {
-        int i = 0;
-        for (VillagerTrades.ITrade[] levelList : tradesLists.values()) {
+        for (int i = 1;i < tradesLists.size() + 1;i++) {
```

Comment:
Just to be sure, is this indexed starting from 1 instead of 0?

Reply:
If you mean the tradesList with indexed, then yes. It starts from 1 because the lowest level a villager can be is lvl 1 and the trades for a level are mapped to the level's number.